### DISCUSSION TITLE
链接 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#58] [General]
https://blog.1l1.icu/links/ 

 欢迎添加本站友链，申请格式及本站信息如下：  名称：夏沫花火zzz🌙 (Muska_Ami)のLife 链接：https://blog.1l1.icu 介绍：Keep doing,keep loving. 头像：https://pan.1l1.icu/f/LG4Te/%E5%A4%B4%E5%83%8F.03.jpeg   要求：  支持HTTPS 支持TLS v1.3 使用受广泛信任的证书 无

**Comment by ltzXiaoYanMo** [comment]
:thinking:

**Comment by Muska-Ami** [comment]
> @ltzXiaoYanMo
> :thinking:

🥺

**Comment by ltzXiaoYanMo** [comment]
> @Muska-Ami
> > @ltzXiaoYanMo
> > :thinking:
> 
> 🥺

~~Markdown 语法忘的差不多了~~

😭

**Comment by ltzXiaoYanMo** [comment]
:thinking:

**Comment by NiuBoss123** [comment]
名称：御坂秋生の小站
链接：https://www.akio.top
介绍：不努力就只能听到别人的好消息
头像：https://server.akio.top/icon

**Comment by Muska-Ami** [comment]
> @NiuBoss123
> 名称：御坂秋生の小站
> 链接：https://www.akio.top
> 介绍：不努力就只能听到别人的好消息
> 头像：https://server.akio.top/icon

加上啦~

**Comment by ltzXiaoYanMo** [comment]
> > @NiuBoss123
> > 名称：御坂秋生の小站
> > 链接：https://www.akio.top
> > 介绍：不努力就只能听到别人的好消息
> > 头像：https://server.akio.top/icon
> 
> 加上啦~

草拟吗加我的就得好几天是罢（

**Comment by Muska-Ami** [comment]
> @ltzXiaoYanMo
> > > @NiuBoss123
> > > 名称：御坂秋生の小站
> > > 链接：https://www.akio.top
> > > 介绍：不努力就只能听到别人的好消息
> > > 头像：https://server.akio.top/icon
> > 
> > 加上啦~
> 
> 草拟吗加我的就得好几天是罢（

谁允许你说话了（

**Comment by ltzXiaoYanMo** [comment]
> > @ltzXiaoYanMo
> > > > @NiuBoss123
> > > > 名称：御坂秋生の小站
> > > > 链接：https://www.akio.top
> > > > 介绍：不努力就只能听到别人的好消息
> > > > 头像：https://server.akio.top/icon
> > > 
> > > 加上啦~
> > 
> > 草拟吗加我的就得好几天是罢（
> 
> 谁允许你说话了（

谁让你开站了（

**Comment by ltzXiaoYanMo** [comment]
名称：KuoHuBlog
链接：https://khbit.cn
介绍：一名Bug带师（不是
头像：https://api.ymbit.cn/images/kuohu_favicon.jpg

**Reply by Muska-Ami** [reply]
加上啦~

**Reply by ltzXiaoYanMo** [reply]
1!5!

**Reply by ltzXiaoYanMo** [reply]
> 加上啦~

喵呜

**Comment by xyyy-cn** [comment]
站点名称：Aehxy的个人小站

站点描述：若知是梦何须醒，不比真如一相会。

站点图标：https://www.aehxy.com/wp-content/uploads/2024/02/avatar.jpg

站点链接：https://www.aehxy.com

**Reply by Muska-Ami** [reply]
加上啦~

**Comment by SRY-CTB** [comment]
备案审核内容不过，帮忙把shanrenyi.top改成blog.shanrenyi.top。

**Reply by Muska-Ami** [reply]
已经改好啦

**Reply by ltzXiaoYanMo** [reply]
> 备案审核内容不过，帮忙把shanrenyi.top改成blog.shanrenyi.top。

我就没见过个人备案备博客能过的（

**Reply by SRY-CTB** [reply]
放了快两年了没啥事，最近新增备案事情来了，就迁走了 呗

**Reply by ltzXiaoYanMo** [reply]
> 放了快两年了没啥事，最近新增备案事情来了，就迁走了 呗

傻逼备案是这样的

**Comment by bingxin666** [comment]
站名：Bingxin Home
简介：好久不见
主页地址：https://byteloid.one
图片：https://blog.byteloid.one/img/march7th.webp

**Reply by Muska-Ami** [reply]
加上啦~

**Comment by MoYuanCN** [comment]
名称：MoYuanCN's Blog
链接：https://MoYuanCN.vip
介绍：如星星般闪耀
头像：https://cdn-fusion.imgimg.cc/i/2024/40b61d0cab175f39.jpg

**Reply by Muska-Ami** [reply]
加上啦~

**Reply by MoYuanCN** [reply]
大概可以删掉了，暂时没在维护了（找时间重新搞一下

**Comment by shiftz300** [comment]
名称: pbr's pole
地址: [pbr.kasetry.cn](https://pbr.kasetry.cn)
介绍: ~~bruh~~ Just another penguin's blog
头像: [pbr.kasetry.cn/upload/pbr.png](https://pbr.kasetry.cn/upload/pbr.png)

**Reply by Muska-Ami** [reply]
卡片不支持 Markdown 语法，请确认介绍信息

**Reply by shiftz300** [reply]
名称: pbr's pole
地址: https://pbr.kasetry.cn
介绍: Just another penguin's blog
头像: https://pbr.kasetry.cn/upload/pbr.png

**Reply by Muska-Ami** [reply]
加上啦~

**Comment by yuhan2680** [comment]
Nya

**Reply by Muska-Ami** [reply]
にゃ~

**Reply by yuhan2680** [reply]
> にゃ~

捉住一只夏沫awa

**Reply by Muska-Ami** [reply]
awa

**Comment by yuhan2680** [comment]
Nya~

**Comment by rxw2023** [comment]
名称：DoctoR’s blog
链接：https://rxw2023-github-io.pages.dev/
介绍：before my body is dry
头像：https://rxw2023-github-io.pages.dev/images/kk.jpg

**Reply by Muska-Ami** [reply]
感谢你的申请，不过能否先将本站加入友链呢？
可以的话，推荐使用一个自己的域名而非 `pages.dev` `github.io` `vercel.app` 这样的公共域名（非强制要求）

**Reply by rxw2023** [reply]
加了友链

**Reply by Muska-Ami** [reply]
加上啦~

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by canmiwu** [comment]
Hi, 欢迎和我做朋友
```yaml
name: 'Canmi' 
intro: '彩笔猫猫' 
link:'https://canmi21.github.io' 
avatar: 'https://canmi21.github.io/img/Canmi.webp'
```

该网站目前挂在 ```bash https://xyy.canmi.xyz/ ```
但是由于计划原因，明年初可能会更换域名，所以建议使用GITHUB.IO来绑定
如果后续有新的域名的话他会和现在一样被重定向到我的域名上，就想现在的
```bash https://xyy.canmi.xyz/ ```，另外可选的包含重定向的链接是 ```bash https://xyy.canmi.xyz/img/Canmi.webp ```


部分的博客框架对重定向的链接(301)十分不友好，如果您正在使用WP作为框架的话推荐使用:
```bash https://raw.githubusercontent.com/Canmi21/Canmi21/refs/heads/main/assets/Canmi.webp ```

另外我现在也会添加你! 大约1-2min后便可在 ```bash https://xyy.canmi.xyz/about/ ``` 看到)~

![](https://xyy.canmi.xyz/img/SSL/20241126190210.png)
![](https://xyy.canmi.xyz/img/SSL/20241126190241.png)
![](https://xyy.canmi.xyz/img/SSL/20241126190301.png)

**Reply by Muska-Ami** [reply]
加上啦~

**Reply by Muska-Ami** [reply]
您的头像似乎有些问题哦
![image](https://github.com/user-attachments/assets/7a776d8d-7683-4bb4-81bb-ba43e8ce4266)

**Reply by canmiwu** [reply]
对哇，所以下面写了，如果你的网站或者浏览器禁止了重定向的链接的话，可能需要使用：https://raw.githubusercontent.com/Canmi21/Canmi21/refs/heads/main/assets/Canmi.webp 或者保存在你自己的网站）（逃）

**Reply by Muska-Ami** [reply]
已修改，不过为其添加了
```
https://proxy-gh.1l1.icu
```
前缀来保证更好的显示效果

**Reply by canmiwu** [reply]
好欸，不过好奇，你的这个代理接口可以传入任何github内的链接嘛

**Reply by Muska-Ami** [reply]
直接访问进去有说明的哦

**Reply by canmi21** [reply]
更新地址: https://arc.canmi.icu/

**Reply by Muska-Ami** [reply]
好了呀~

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by Ataidawa** [comment]
站点名称： 懒事屋
站点地址： https://ataida.us.kg/
站点描述： 就算不是理工科也要耍博客！
站点头像： https://ataida.us.kg/images/avatar.jpg

**Reply by Muska-Ami** [reply]
忙着忙着忘记了，已为您添加友链~

**Reply by Ataidawa** [reply]
> 忙着忙着忘记了，已为您添加友链~

救命，域名不知道怎么掉了…… 保险起见大大还是挂Github.io吧 https://ataidawa.github.io/

**Reply by Muska-Ami** [reply]
已经改啦~

**Reply by Muska-Ami** [reply]
可以考虑我的免费域名服务，具体看链接页面最顶部哦（x

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by Muska-Ami** [comment]
您好，您的博客文章数较少，我会暂时观察一段时间，确保长期开设，避免本站出现死链降低权重

From Muska Ami（夏沫花火zzz🌙）
________________________________

**Comment by lxhtt** [comment]
网站名称：落雪の自留地  
网站地址：https://www.lxhtt.cn
网站图标：https://www.lxhtt.cn/img/avatar.webp
网站简介：衣带渐宽终不悔，为伊消得人憔悴。

**Reply by Muska-Ami** [reply]
加上啦~

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by ZeroWolf233** [comment]
🤔

**Comment by Daiyangcheng** [comment]
链接: https://www.daiyangcheng.cn/
头像: https://cravatar.cn/avatar/5417982dfeabcb9a22bfc582c88a4759?s=2000
站点名称: Daiyangcheng‘s Blog
描述：dyc 的碎碎念

**Reply by Muska-Ami** [reply]
加上啦～

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by Shiroiame-Kusu** [comment]
网站名称：楠ちゃん の Mirror
网站链接：https://blog.nyat.icu/
链接图片：https://cdn.nyat.icu/image/siteicon.png
网站描述：啊，你在问我吗？

**Reply by Muska-Ami** [reply]
加上啦～

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by ADAIBLOG** [comment]
名称: 阿呆
链接: https://adai.fun
介绍: 分享，学习有趣的知识
头像: https://www.adai.fun/img/adai.png

**Reply by Muska-Ami** [reply]
加上啦~

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by SALTWOOD** [comment]
名称：盐木的小破窝
描述：卷土重来未可知。——杜牧《题乌江亭》
地址：https://ski.ink/
头像：https://saltwood.top:5244/d/友链头像/盐木.jpg（不建议直接使用，下载下来上什么的比较好，非要用链接建议用 https://ski.ink/favicon.ico）

花火小可爱🥰

**Reply by Muska-Ami** [reply]
加上啦~

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by ADAIBLOG** [comment]


**Reply by canmi21** [reply]
![image](https://github.com/user-attachments/assets/a52e9a43-57f5-44a4-a49a-587c80d07d15)
您是不是引用错了，消息提醒提示到我来了，如果是指[我的网站](https://arc.canmi.icu/)的话那么确实，众所周知的原因，clouldflare的解析在国内移动用户基本不可用，电信联通一般可以正常连接。

**Comment by ADAIBLOG** [comment]
更新一下新的地址
- name: 阿呆
  link: https://adaiblog.github.io
  avatar: https://q2.qlogo.cn/headimg_dl?dst_uin=1432231192&spec=640
  descr: 分享生活中的事物

**Reply by Muska-Ami** [reply]
更新好啦~

**Comment by worable233** [comment]
本站信息
站点名称：worable's Blog
站点地址：https://www.worable.top/
LOGO链接：https://www.worable.top/wp-content/uploads/2026/02/2026020109015019.jpg
站点描述：≡ω≡ 嗯对，一名普通的职一牲~

ref(APA): 唔嘫worable.worable&#039;s Blog.https://www.worable.top. Retrieved 2/21/2026.

**Reply by Muska-Ami** [reply]
加上啦~
最近忙于学业回复的有点晚抱歉喵

**Reply by Muska-Ami** [reply]
Ciallo\~麻烦更新一下友链，因为 icu 管局把我的域名改成溢价域名了，只好换了一个，原本的 1l1.icu -> amaicat.work ，感谢~

**Comment by yuhan2680** [comment]
更新一下我的个人网站地址
naiwenel.com

**Reply by Muska-Ami** [reply]
嗯...没找到对应的友链记录，如果需要的话麻烦重新填写信息申请一下，感谢~

**Comment by DarkDijun** [comment]
泥嚎博主，我来加友链啦~
博客名称：Damon_ZhangのBlog
博客地址：https://blog.damonz.cn/
博客描述：于万千星光沉溺
博客头像：https://blog.damonz.cn/upload/avatar.jpg
已添加贵站，有空多来玩儿~

**Reply by Muska-Ami** [reply]
非常抱歉这么晚才回复，目前看好像没有添加诶？

**Comment by shiftz300** [comment]
名称：maincore
链接：[https://maincore.work](https://maincore.work)
介绍：N/A
头像：暂用 [https://s2.loli.net/2024/06/22/2DuQoJMtd1GjfcP.png](https://s2.loli.net/2024/06/22/2DuQoJMtd1GjfcP.png)

**Comment by ChenQihan666** [comment]
来申请一个友链：
网站名称： 启涵的小破站
网站地址： https://qihanx.cn
网站图标： https://qihanx.cn/favicon.png
网站简介： 最慢的步伐不是跬步，而是徘徊；最快的脚步不是冲刺，而是坚持。
